### DISCUSSION TITLE
Internal Error 500

### DISCUSSION THREAD

**Discussion by VarchasvH** [EddieHubCommunity/LinkFree#5068] [Q&A] [answered]
When I am trying to open my LinkFree page, it's showing Internal Server Error: 500, Any Solutions?
![image](https://user-images.githubusercontent.com/100064742/221367788-97a1496d-7429-413b-97f5-10a54185fdd5.png)

**Comment by swagataroy30** [comment]
![Screenshot from 2023-02-28 23-47-21](https://user-images.githubusercontent.com/116095538/221943811-ff8e8af1-ba32-4a3b-8776-049d6e54460b.png)

Hey, I did look into it - you need to change "Socials"  key to "links" - which is causing the error . This might have been overlooked while reviewing the PR ... you can maybe make a PR to update your json file accordingly. You can refer to this [docs](https://linkfree.eddiehub.io/docs/quickstart) for further reference. Thank You!

**Reply by VarchasvH** [reply]
> ![Screenshot from 2023-02-28 23-47-21](https://user-images.githubusercontent.com/116095538/221943811-ff8e8af1-ba32-4a3b-8776-049d6e54460b.png)
> 
> Hey, I did look into it - you need to change "Socials" key to "links" - which is causing the error . This might have been overlooked while reviewing the PR ... you can maybe make a PR to update your json file accordingly. You can refer to this [docs](https://linkfree.eddiehub.io/docs/quickstart) for further reference. Thank You!

Thank you, I made a PR and now I am waiting for it to go live.

**Comment by Ananya2001-an** [comment, marked as answer]
Hey @VarchasvH there are actually both `socials` and `links` as options for you to add in your LinkFree JSON profile. Just remember next time that if you add socials it will just have url and icon props.

**Reply by VarchasvH** [reply]
Ok, Thanks for the help.
